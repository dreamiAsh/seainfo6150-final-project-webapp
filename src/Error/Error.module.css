.box{
    font-size: 60px;
    color: grey;
    width: 100%;
    text-align: center;
    margin: 150px 0 300px;
}