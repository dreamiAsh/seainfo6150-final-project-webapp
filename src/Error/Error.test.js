import React from "react";
import { render, fireEvent } from "@testing-library/react";
import Error from "./Error.jsx";

it("renders the Error component correctly", () => {
  const { container } = render(<Error />);
  expect(container).toMatchSnapshot();
});