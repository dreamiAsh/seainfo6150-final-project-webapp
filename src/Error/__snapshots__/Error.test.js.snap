// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders the Error component correctly 1`] = `
<div>
  <div
    class="box"
  >
    404 Not found
  </div>
</div>
`;
